src/main/java/org/opentripplanner/graph_builder/module/geometry/GeometryAndBlockProcessor.java

```diff
@@ -308,6 +311,7 @@ private LineString[] createGeometry(FeedScopedId shapeId, List<StopTime> stopTim
       // this only happens on shape which have points very far from
       // their stop sequence. So we'll fall back to trivial stop-to-stop
       // linking, even though theoretically we could do better.
+      issueStore.add(new ShapeGeometryTooFar(stopTimes.get(0).getTrip().getId(), shapeId));
```

Comment:
It would be nice to get information from `getLinearLocations` and `getStopLocations` on which stop it failed to find the shape segment. Also in the error, and its HTML part, we could include a link to the visualisation of the route + stops